bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/AVMFritzBaseBridgeHandler.java

```diff
@@ -140,13 +141,26 @@ public void initialize() {
         }
 
         if (configValid) {
-            this.connection = new FritzAhaWebInterface(config, this, httpClient);
+            updateStatus(ThingStatus.UNKNOWN);
+            initConnections();
+        }
+    }
 
+    protected synchronized void initConnections() {
+        AVMFritzBoxConfiguration config = getConfigAs(AVMFritzBoxConfiguration.class);
+        if (this.connection == null) {
+            this.connection = new FritzAhaWebInterface(config, this, httpClient);
             stopPolling();
             startPolling();
         }
     }
 
+    @Override
+    public void channelLinked(ChannelUID channelUID) {
```

Comment:
Should we dispose the `CallMonitor` if all call monitor channels are unlinked?

Reply:
done.